src/Transitions/Transition.js

```diff
@@ -0,0 +1,173 @@
+'use strict';
+import React from 'react';
+import TransitionEvents from '../utils/TransitionEvents';
+import classnames from 'classnames';
+
+function omit(obj, keys) {
+  let included = Object.keys(obj).filter( k => keys.indexOf(k) === -1)
+  let newObj = {};
+
+  included.forEach( key => newObj[key] = obj[key] )
+  return newObj;
+}
```

Comment:
Agree. :cherries: 
`lodash`-refactoring could be done in the far future if an issue arise.